docs/user-guide.md

```diff
@@ -260,10 +261,10 @@ For API details, refer to the [API docs](api/scala/index.html#org.graphframes.Gr
 
 {% highlight scala %}
 import org.graphframes.examples
-val g: GraphFrame = examples.Graphs.friends  // get example graph
+val g = examples.Graphs.friends  // get example graph
 
 // Search for pairs of vertices with edges in both directions between them.
-val motifs: DataFrame = g.find("(a)-[e]->(b); (b)-[e2]->(a)")
```

Comment:
I think it's kinda useful to see the data type in doc text in somewhat unintuitive cases, like this one

Reply:
Ok, I've removed it because with current import, it won't work. I can update the code to return types...